### ISSUE TITLE
Softmax

### ISSUE THREAD

**Issue by ilkarman** [ilkarman/DemoNeuralNet#3]
Add multinomial logistic regression example to bridge gap between logit and NN
